acapy_plugin_toolbox/connections.py

```diff
@@ -324,6 +324,7 @@ async def handle(self, context: RequestContext, responder: BaseResponder):
             invitation,
             auto_accept=context.message.auto_accept,
             mediation_id=context.message.mediation_id,
+            connection_reuse = context.message.use_existing_connection,
```

Comment:
This needs to be on the ReceiveOOBInvitationHandler rather than on the ReceiveInvitationHandler :slightly_smiling_face: